src/django/pfb_analysis/views.py

```diff
@@ -70,6 +70,7 @@ def results(self, request, pk=None):
 class NeighborhoodMixin(APIView):
     """Shared properties of the neighborhood viewsets."""
 
+    queryset = Neighborhood.objects.all()
```

Comment:
I wonder if that was fallout from the ordering.  Or did you get the crash even with the mixin first?  Anyway yeah, this seems to be working.